client/coral-admin/src/containers/Configure/Configure.css

```diff
@@ -77,26 +80,28 @@
 }
 
 .charCountTexfieldEnabled {
-  border-color: #4caf50;
+  border-color: #00796b;
 }
 
 .charCountTexfield:focus {
   outline: none;
 }
 
 .changedSave {
-  background-color:#4caf50;
+  background-color: #00796B;
+  color: white;
 }
 
 .copiedText {
-  color: #008000;
+  color: #00796b;
```

Comment:
Yes, we will add a file with all of our CSS Vars but this will not enter in this PR.